### ISSUE TITLE
Install error: "fatal: Not a git repository (or any of the parent directories): .git"

### ISSUE THREAD

**Issue by vstepaniuk** [slytomcat/yandex-disk-indicator#196]
Describe what happened with indicator
"fatal: Not a git repository (or any of the parent directories): .git" error during the installation process according to the instructions in `README.md`.

Describe steps to recreate the issue

1. Download an archive from https://github.com/slytomcat/yandex-disk-indicator/releases/latest.
2. Extract the files and change to `build` directory.
3. Run `sudo ./install.sh`.

Make debugging log:
```
sudo ./install.sh 
yd-tools (1.10.3) artful; urgency=medium
  * Changes:
    - file watcher simplified (-iNotify +stat+timer)
    - multiselection in FileChooserDialog (while adding exluded folders)

 -- Sly_tom_cat <slytomcat@mail.ru>  Sun, 20 May 2018 11:44:32 +0300
fatal: Not a git repository (or any of the parent directories): .git
```

Describe your OS (distribution, version, desktop environment)
Ubuntu 17.10 x64

The reason is `git clean -df .` line in `clean.sh`, which is unnecessary if the program was downloaded as an archive.

**Comment by slytomcat** [comment]
Yes, I see. 
It is incorrect to use 'git clean' in install script. 

I'll change it.

**Comment by vstepaniuk** [comment]
Ok, thanks!

**Comment by slytomcat** [comment]
fixed in [1.10.4](https://github.com/slytomcat/yandex-disk-indicator/releases/tag/1.10.4)
